### PR TITLE
Bc1 22 home page section show land

### PR DESCRIPTION
**Pull request by sadiGFG**

### CODE REVIEW

File: shared/scss/base/_reset.scss [hunk 1]
```diff
@@ -25,7 +25,6 @@ picture,
 svg,
 video {
   display: block;
-  width: 100%;
```
Comment (abdolmajed3) [inline shared/scss/base/_reset.scss:28]: this file is currently active in our projects. 
I think if this line of codes affect on some of your code you can add some changes in your own code to disable it in your code.
what do you think?

File: shared/scss/components/_button.scss
Comment (abdolmajed3) [inline shared/scss/components/_button.scss:1]: i cant find any changes here. except of one enter in the start of the fie ;)

File: show-land/sass/layout/_top-nav.scss [hunk 2]
```diff
@@ -1,31 +1,26 @@
-.container {
-  .header {
+.main-nav {
+  &__list {
     display: flex;
-    justify-content: space-between;
+    padding-top: 1rem;
+    gap: 1.5rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_top-nav.scss:5]: you can use from our shared standard codes.
1- import shared variables in your main codes.
2- add this code in the start of your current file : 
`@use "sass:list"`
then use this code :
`gap: list.nth($spaces, 6)
`

File: show-land/sass/layout/_top-nav.scss [hunk 3]
```diff
@@ -1,31 +1,26 @@
...
+    font-weight: 700;
   }
 }
-
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+.logo {
+  padding-top: 0.8rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_top-nav.scss:23]: use our shared spacing.

**Review by MaxRichard2007 — APPROVED** [review]

**Review by abdolmajed3 — APPROVED** [review]

**Pull request merged**